test/run-functional-tests

```diff
@@ -190,7 +190,7 @@ deploy_ha_node() {
         else
             podman_security=""
         fi
-	make_cmd="cd $PROJECT_INSIDE;./autogen.sh && ./configure --prefix /usr && make install && make install-crmconfDATA prefix= && cp /usr/bin/crm /usr/sbin"
+	make_cmd="cd $PROJECT_INSIDE;./autogen.sh && ./configure --prefix /usr && make all-local && make install && make install-crmconfDATA prefix= && cp /usr/bin/crm /usr/sbin"
```

Comment:
If you need to run `make all-lcoal` before `make install`, your target dependencies in Makefile is broken.